pbiviz.json

```diff
@@ -1,33 +1,23 @@
 {
   "visual": {
     "name": "textFilter",
-    "displayName": "Text Filter",
-    "guid": "textFilter25A4896A83E0487089E2B90C9AE57C8A",
+    "displayName": "TextFilter",
```

Comment:
You'll need to copy over `displayName`, `guid`, `version`, `description` from the old one.

Reply:
Fixed